src/shared-components/dialogModal/index.tsx

```diff
@@ -113,6 +123,7 @@ export const DialogModal = ({
 
 DialogModal.propTypes = {
   children: PropTypes.node.isRequired,
+  modalColor: PropTypes.oneOf([primaryTheme.COLORS.background]),
```

Comment:
We'll also want to add the `secondaryTheme` version:

`modalColor: PropTypes.oneOf([primaryTheme.COLORS.background, secondaryTheme.COLORS.background]),`